sample-app/src/utils/geolocationutils.tsx

```diff
@@ -0,0 +1,56 @@
+import { AnswersActions } from "@yext/answers-headless-react";
```

Comment:
I don't think this file needs to be .tsx and can just be .ts. Maybe we could change the other "blah-utils.tsx" files too? I'm also not a huge fan of naming everything "blah-utils" anymore since the parent folder is already called "utils" and the naming is a bit vague, but that's out of scope here.

Reply:
hm yea, will change the file extension. As for the name, tom suggested `SearchHandler` to change this into a class, so I will probably name the file along that line without 'util'